src/locales/en.json

```diff
@@ -501,7 +501,8 @@
       "overlappingStarted": "The start time is within the period of an existing and reviewed shift.",
       "overlappingStopped": "The end time is within the period of an already existing and reviewed shift.",
       "eightTwentyRule": "As a rule, student workers should not be obliged to work before 08:00 and after 20:00.",
-      "autoworking timeCutting": "Mandatory breaktimes are missing on this workday. Clock will subtract the legally required breaktimes automatically from your working time."
+      "autoworking timeCutting": "Mandatory breaktimes are missing on this workday. Clock will subtract the legally required breaktimes automatically from your working time.",
+      "month_already_locked": "This Month is already locked. Therefor there are no new Shifts allowed."
```

Comment:
This month has already been locked. You cannot save new shifts within this month.